skills/csharp/emailskill/Services/BotServices.cs

```diff
@@ -27,24 +27,31 @@ public BotServices(BotSettings settings, IBotTelemetryClient client)
                 var config = pair.Value;
 
                 var telemetryClient = client;
-                var luisOptions = new LuisPredictionOptions()
-                {
-                    TelemetryClient = telemetryClient,
-                    LogPersonalInformation = true,
-                };
+
+                LuisRecognizerOptionsV3 luisOptions;
 
                 if (config.DispatchModel != null)
                 {
                     var dispatchApp = new LuisApplication(config.DispatchModel.AppId, config.DispatchModel.SubscriptionKey, config.DispatchModel.GetEndpoint());
-                    set.DispatchService = new LuisRecognizer(dispatchApp, luisOptions);
+                    luisOptions = new LuisRecognizerOptionsV3(dispatchApp)
+                    {
+                        TelemetryClient = telemetryClient,
+                        LogPersonalInformation = true,
+                    };
+                    set.DispatchService = new LuisRecognizer(luisOptions);
                 }
 
                 if (config.LanguageModels != null)
                 {
                     foreach (var model in config.LanguageModels)
                     {
                         var luisApp = new LuisApplication(model.AppId, model.SubscriptionKey, model.GetEndpoint());
-                        set.LuisServices.Add(model.Id, new LuisRecognizer(luisApp, luisOptions));
+                        luisOptions = new LuisRecognizerOptionsV3(luisApp)
+                        {
+                            TelemetryClient = telemetryClient,
+                            LogPersonalInformation = true,
```

Comment:
should add 
`PredictionOptions = new Microsoft.Bot.Builder.AI.LuisV3.LuisPredictionOptions() { IncludeInstanceData = true }`
for _instance is closed by default in luisV3 but the email use it.

Reply:
thanks for pointing out the issue. fixed.